common-client/src/main/scala/org/genivi/sota/core/FakeDeviceRegistry.scala

```diff
@@ -129,4 +129,8 @@ class FakeDeviceRegistry(namespace: Namespace)
   def addGroup(group: Uuid, devices: Seq[Uuid]): Unit = {
     groups.put(group, devices)
   }
+
+  def setInstalledPackages
+  (device: Uuid, json: io.circe.Json)(implicit ec: ExecutionContext) : Future[Unit] =
```

Comment:
The json isn't a list, but a json object with a list of firmware and a list of software. I could move the 'InstalledSoftware' object from resolver into common and use it in core also?